_includes/css/style.css

```diff
@@ -131,6 +131,28 @@ a:focus {
 	border-color: transparent;
 }
 
+.navbar-toggle {
+	border: none;
+	margin: 0;
+	padding: 18px 26px 18px 11px;
+}
+
+.navbar-toggle .icon-bar {
+	background-color: {{ site.colors.link }};
```

Comment:
Use CSS variables

Reply:
Since we don't have sass/less integrated, you can [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for color/background color etc.

Reply:
@bhansa  Thank you for the feedback.

We can do this, but these constants are used in other instructions [in the same file](/HydPy/HydPy/blob/master/_includes/css/style.css). How do they work now?

For example:
![image](https://user-images.githubusercontent.com/3881568/66399449-e75b2600-e9df-11e9-9805-f8c5a1dc666c.png)

Reply:
Ah, that's bad. I didn't notice that before, thanks for the info. 
I'll try to use these variables from a sass file instead of config as part of #42

Reply:
What is the best way for me to do now in order to simplify the work you are planning?
I'm afraid that if I replace `{{ site.colors.link }}` and `{{ site.colors.link }}` with the colors from the configuration file or with some CSS-variables, I will only complicate the upcoming code upgrade.

Reply:
Ignore css variables for now, we will go ahead with this change.